diff --git a/node_modules/react-native-calendars/src/calendar/day/period/index.js b/node_modules/react-native-calendars/src/calendar/day/period/index.js
index fc0187d..84f2cf2 100644
--- a/node_modules/react-native-calendars/src/calendar/day/period/index.js
+++ b/node_modules/react-native-calendars/src/calendar/day/period/index.js
@@ -4,6 +4,8 @@ import { TouchableWithoutFeedback, TouchableOpacity, Text, View } from 'react-na
 import { xdateToData } from '../../../interface';
 import styleConstructor from './style';
 import Dot from '../dot';
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen';
+import { colors } from '../../../../../../src/styles/Colors'
 const PeriodDay = (props) => {
     const { theme, marking, date, onPress, onLongPress, state, accessibilityLabel, testID, children } = props;
     const dateData = date ? xdateToData(date) : undefined;
@@ -105,12 +107,21 @@ const PeriodDay = (props) => {
         }
         return { leftFillerStyle, rightFillerStyle, fillerStyle };
     }, [marking]);
+
     const renderFillers = () => {
         if (marking) {
             return (<View style={[style.current.fillers, fillerStyles.fillerStyle]}>
-          <View style={[style.current.leftFiller, fillerStyles.leftFillerStyle]}/>
-          <View style={[style.current.rightFiller, fillerStyles.rightFillerStyle]}/>
-        </View>);
+                <View style={[style.current.leftFiller, fillerStyles.leftFillerStyle]} />
+                {marking.startingDay && marking.color != colors.dark_blue1_color ? <View style={{ backgroundColor: colors.white_color, marginLeft: wp(3.6), borderTopLeftRadius: wp(5), borderBottomLeftRadius: wp(5) }}>
+                    <View style={[style.current.dayStyles, style.current.roundBorder, { left: wp(-1.5) }]} />
+                </View> : null}
+                <View style={[style.current.rightFiller, fillerStyles.rightFillerStyle]} />
+                {marking.endingDay && marking.color != colors.dark_blue1_color ? <View style={{ backgroundColor: colors.white_color, marginRight: wp(3.8), borderBottomRightRadius: wp(5), borderTopRightRadius: wp(5) }}>
+                    <View style={[style.current.dayStyles, style.current.roundBorder, { left: wp(2) }]} />
+                </View>
+                    : null}
+
+            </View>);
         }
     };
     const _onPress = useCallback(() => {
@@ -121,15 +132,15 @@ const PeriodDay = (props) => {
     }, [onLongPress]);
     const Component = marking ? TouchableWithoutFeedback : TouchableOpacity;
     return (<Component testID={testID} onPress={_onPress} onLongPress={_onLongPress} disabled={marking?.disableTouchEvent} accessible accessibilityRole={marking?.disableTouchEvent ? undefined : 'button'} accessibilityLabel={accessibilityLabel}>
-      <View style={style.current.wrapper}>
-        {renderFillers()}
-        <View style={containerStyle}>
-          <Text allowFontScaling={false} style={textStyle}>
-            {String(children)}
-          </Text>
-          <Dot theme={theme} color={marking?.dotColor} marked={marking?.marked}/>
+        <View style={style.current.wrapper}>
+            {renderFillers()}
+            <View style={containerStyle}>
+                <Text allowFontScaling={false} style={textStyle}>
+                    {String(children)}
+                </Text>
+                <Dot theme={theme} color={marking?.dotColor} marked={marking?.marked} />
+            </View>
         </View>
-      </View>
     </Component>);
 };
 export default PeriodDay;
diff --git a/node_modules/react-native-calendars/src/calendar/day/period/style.js b/node_modules/react-native-calendars/src/calendar/day/period/style.js
index 465c11e..26d65c6 100644
--- a/node_modules/react-native-calendars/src/calendar/day/period/style.js
+++ b/node_modules/react-native-calendars/src/calendar/day/period/style.js
@@ -1,5 +1,7 @@
 import { StyleSheet } from 'react-native';
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen';
 import * as defaultStyle from '../../../style';
+import { colors } from '../../../../../../src/styles/Colors'
 const FILLER_HEIGHT = 34;
 export default function styleConstructor(theme = {}) {
     const appStyle = { ...defaultStyle, ...theme };
@@ -24,7 +26,7 @@ export default function styleConstructor(theme = {}) {
         leftFiller: {
             backgroundColor: appStyle.calendarBackground,
             height: FILLER_HEIGHT,
-            flex: 1
+            flex: 1,
         },
         rightFiller: {
             backgroundColor: appStyle.calendarBackground,
@@ -53,7 +55,17 @@ export default function styleConstructor(theme = {}) {
             color: appStyle.textDisabledColor
         },
         inactiveText: {
-            color: appStyle.textInactiveColor
+            color: appStyle.textInactiveColor,
+        },
+        dayStyles: {
+            width: wp(9),
+            height: wp(9),
+            borderRadius: wp(9),
+        },
+        roundBorder: {
+            backgroundColor: colors.dark_blue1_color,
+            borderWidth: wp(0.5),
+            // borderColor: colors.calendar_round_border,
         },
         // @ts-expect-error
         ...(theme['stylesheet.day.period'] || {})
diff --git a/node_modules/react-native-calendars/src/calendar/index.js b/node_modules/react-native-calendars/src/calendar/index.js
index 61134c9..4adff19 100644
--- a/node_modules/react-native-calendars/src/calendar/index.js
+++ b/node_modules/react-native-calendars/src/calendar/index.js
@@ -90,21 +90,21 @@ const Calendar = (props) => {
     }, [onSwipeLeft, onSwipeRight]);
     const renderWeekNumber = (weekNumber) => {
         return (<View style={style.current.dayContainer} key={`week-container-${weekNumber}`}>
-        <BasicDay key={`week-${weekNumber}`} marking={weekNumberMarking.current} 
-        // state='disabled'
-        theme={theme} testID={`${WEEK_NUMBER}-${weekNumber}`}>
-          {weekNumber}
-        </BasicDay>
-      </View>);
+            <BasicDay key={`week-${weekNumber}`} marking={weekNumberMarking.current}
+                // state='disabled'
+                theme={theme} testID={`${WEEK_NUMBER}-${weekNumber}`}>
+                {weekNumber}
+            </BasicDay>
+        </View>);
     };
     const renderDay = (day, id) => {
         const dayProps = extractDayProps(props);
         if (!sameMonth(day, currentMonth) && hideExtraDays) {
-            return <View key={id} style={style.current.emptyDayContainer}/>;
+            return <View key={id} style={style.current.emptyDayContainer} />;
         }
         return (<View style={style.current.dayContainer} key={id}>
-        <Day {...dayProps} date={toMarkingFormat(day)} state={getState(day, currentMonth, props)} marking={markedDates?.[toMarkingFormat(day)]} onPress={onPressDay} onLongPress={onLongPressDay}/>
-      </View>);
+            <Day {...dayProps} date={toMarkingFormat(day)} state={getState(day, currentMonth, props)} marking={markedDates?.[toMarkingFormat(day)]} onPress={onPressDay} onLongPress={onLongPressDay} />
+        </View>);
     };
     const renderWeek = (days, id) => {
         const week = [];
@@ -115,8 +115,8 @@ const Calendar = (props) => {
             week.unshift(renderWeekNumber(days[days.length - 1].getWeek()));
         }
         return (<View style={style.current.week} key={id}>
-        {week}
-      </View>);
+            {week}
+        </View>);
     };
     const renderMonth = () => {
         const shouldShowSixWeeks = showSixWeeks && !hideExtraDays;
@@ -141,7 +141,7 @@ const Calendar = (props) => {
         const ref = customHeader ? undefined : header;
         const CustomHeader = customHeader;
         const HeaderComponent = customHeader ? CustomHeader : CalendarHeader;
-        return (<HeaderComponent {...headerProps} testID={testID} style={headerStyle} ref={ref} month={currentMonth} addMonth={addMonth} displayLoadingIndicator={shouldDisplayIndicator}/>);
+        return (<HeaderComponent {...headerProps} testID={testID} style={headerStyle} ref={ref} month={currentMonth} addMonth={addMonth} displayLoadingIndicator={shouldDisplayIndicator} />);
     };
     const GestureComponent = enableSwipeMonths ? GestureRecognizer : View;
     const swipeProps = {
@@ -149,12 +149,12 @@ const Calendar = (props) => {
     };
     const gestureProps = enableSwipeMonths ? swipeProps : undefined;
     return (<GestureComponent {...gestureProps}>
-      <View style={[style.current.container, propsStyle]} accessibilityElementsHidden={accessibilityElementsHidden} // iOS
-     importantForAccessibility={importantForAccessibility} // Android
-    >
-        {renderHeader()}
-        {renderMonth()}
-      </View>
+        <View style={[style.current.container, propsStyle]} accessibilityElementsHidden={accessibilityElementsHidden} // iOS
+            importantForAccessibility={importantForAccessibility} // Android
+        >
+            {renderHeader()}
+            {renderMonth()}
+        </View>
     </GestureComponent>);
 };
 export default Calendar;
