diff --git a/node_modules/react-native-gifted-chat/lib/Models.d.ts b/node_modules/react-native-gifted-chat/lib/Models.d.ts
index 78539c7..4dc2516 100644
--- a/node_modules/react-native-gifted-chat/lib/Models.d.ts
+++ b/node_modules/react-native-gifted-chat/lib/Models.d.ts
@@ -38,18 +38,21 @@ export interface QuickReplies {
     keepIt?: boolean;
 }
 export interface IMessage {
-    _id: string | number;
-    text: string;
+    _id?: string | number;
+    text?: string;
     createdAt: Date | number;
     user: User;
-    image?: string;
-    video?: string;
+    image?: ImageList;
+    video?: VideoList;
     audio?: string;
     system?: boolean;
     sent?: boolean;
     received?: boolean;
     pending?: boolean;
     quickReplies?: QuickReplies;
+    type?: string;
+    jobData?: JobDataProps
+    attachment?: DocList
 }
 export declare type IChatMessage = IMessage;
 export interface MessageVideoProps<TMessage extends IMessage> {
